WooCommerce/Classes/ViewModels/Order Details/OrderDetailsViewModel.swift

```diff
@@ -894,3 +922,12 @@ private extension OrderDetailsViewModel {
             comment: "Notice that appears when no receipt can be retrieved upon tapping on 'See receipt' in the Order Details view.")
     }
 }
+
+// MARK: - Constants
+private extension OrderDetailsViewModel {
+    enum Constants {
+        /// Minimum version of Woo Shipping extension required for app support.
+        /// This should be updated to 1.0.6 once that version is released.
```

Comment:
We will have a pre-release project task to make this change, per this discussion: p1725031043052749-slack-C02KUCFCSFP